### DISCUSSION TITLE
Ability to give a name to a widget that is being "Save as Default" which allows several defaults for one same widget

### DISCUSSION THREAD

**Discussion by oscarhenao** [elementor/elementor#23262] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

I can only have one default for each saved widget

### Describe the solution you'd like

The "Save as Default" feature for elements introduced as stable on v3.11.0 could be enhanced to allow the user to give a name to a saved as default element. This could allow to have several defaults for one same saved element. Something like, let's say for the Image element to have defaults for "Large image", "Medium Image", and "Small image".

**Comment by foolstuck** [comment]
Would love to have the feature. Alternatively, you can make a section template with the widget configured, and each time you need the widget, just import it.
